lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2741,55 +2744,78 @@ private static function get_block_nodes( $theme_json, $selectors = array(), $opt
 		}
 
 		foreach ( $theme_json['styles']['blocks'] as $name => $node ) {
-			$selector = null;
-			if ( isset( $selectors[ $name ]['selector'] ) ) {
-				$selector = $selectors[ $name ]['selector'];
-			}
-
-			$duotone_selector = null;
-			if ( isset( $selectors[ $name ]['duotone'] ) ) {
-				$duotone_selector = $selectors[ $name ]['duotone'];
-			}
+			$include_node_paths_only = $options['include_node_paths_only'] ?? false;
```

Comment:
See the minor updates made to this in the finally committed version: https://core.trac.wordpress.org/changeset/59213

(Only about moving these assignments to before the loop.)

Reply:
Excellent. I'll make the change. Thank you!